### DISCUSSION TITLE
Looking for 64 bit dll of WPFLocalizeExtension.dll for our 64 bit application

### DISCUSSION THREAD

**Discussion by susmitakanchinadham** [SeriousM/WPFLocalizationExtension#349] [Q&A] [answered]
I'm looking for 64 bit dll of WPFLocalizeExtension.dll for our 64 bit application.
Could you please point me to them.
Thanks!

**Comment by UeliHilfiker** [comment, marked as answer]
I think it should already compiled as "AnyCPU", so you can use the same dll for 32 bit and for 64 bit applications.
